.changeset/mighty-turkeys-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@my-sahab/mui': patch
+---
+
+feat: pass adapterLocale to adapter
```

Comment:
feat: add "adapterLocale" property to "localeOptions" interface in "MultiLocalizationProvider"